components/core/src/grammar/kibana/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+ANTLR_TARGET(KibanaParser
+        Kibana.g4
+        LEXER PARSER VISITOR
+        PACKAGE kibana
+        )
+
+# Expose location of headers for generated lexer/parser/visitor to parent
+# CMakeLists file
+set(GENERATED_KIBANA_INCLUDE_DIR ${ANTLR_KibanaParser_OUTPUT_DIR} PARENT_SCOPE)
```

Comment:
Fwiw, I think there is a cleaner way to do this using `INTERFACE` and `target_include_directories`, but I have yet to really look into it myself yet and don't think it is necessary for this PR.
If my understanding is correct, it should be similar to how antlr is exposing its generated headers.
This would mean that a target linking with your `Kibana` target would automatically get the include directories, as opposed to you needing to add them yourself through the `include_directories` call with `GENERATED_KIBANA_INCLUDE_DIR`.

If you're interested in trying it a bit I think this link begins explaining things:
https://kubasejdak.com/modern-cmake-is-like-inheritance

Reply:
I looked into it more and my understanding is the fix should be:
```
target_include_directories(Kibana
    INTERFACE
    ${ANTLR_KibanaParser_OUTPUT_DIR}
    )
```
(and removing `GENERATED_KIBANA_INCLUDE_DIR`)

Can you add it to the PR (or let me know if there are issues)?

Reply:
Yeah, this seems to work fine